### PR TITLE
Use Meta content provider in tcib

### PR DESCRIPTION
**Pull request by raukadah**
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1892

### CODE REVIEW

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
Successful run: https://review.rdoproject.org/zuul/buildset/3787fa2b2e4d4acdafa5aa817ace0b76

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
Successful run: https://review.rdoproject.org/zuul/buildset/1c9a799f9bcf4b28adc80bf4e2a6bb7d

File: zuul.d/job.yaml [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 - job:
-    name: tcib-build-containers
-    parent: cifmw-tcib-base
-    nodeset: centos-stream-9-vexxhost
-    vars:
-      cifmw_build_containers_registry_namespace: podified-antelope-centos9
-      cifmw_build_containers_image_tag: "{{ cifmw_repo_setup_full_hash }}"
+    name: tcib-podified-multinode-edpm-deployment-crc
+    parent: podified-multinode-edpm-deployment-crc
+    dependencies: ["openstack-meta-content-provider"]
```
Comment (marios) [inline zuul.d/job.yaml:5]: maybe better to use the explicit list syntax here for readability?
dependencies:
    - openstack-meta-content-provider

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -3,8 +3,20 @@
...
+              - .ci-operator.yaml
+              - .pre-commit-config.yaml
+              - .pylintrc
+              - .stestr.conf
+              - .github/workflows
+              - .coveragerc
+              - LICENSE
+              - OWNERS*
+        - tcib-crc-podified-edpm-baremetal
+        - tcib-podified-multinode-edpm-deployment-crc
```
Comment (marios) [inline zuul.d/projects.yaml:22]: did you mean to add 
```
irrelevant-files: *if 
```
on these

File: zuul.d/job.yaml (hunk 1 above)
Comment (raukadah) [reply zuul.d/job.yaml]: Done

File: zuul.d/projects.yaml (hunk 2 above)
Comment (raukadah) [reply zuul.d/projects.yaml]: Done

File: zuul.d/job.yaml [hunk 3]
```diff
@@ -20,29 +18,24 @@
...
-    irrelevant-files: *if
     vars: &edpm_vars
-      cifmw_update_containers_registry: "{{ content_provider_registry_ip }}:5001"
-      cifmw_update_containers_org: "podified-antelope-centos9"
-      cifmw_update_containers_tag: "{{ content_provider_dlrn_md5_hash }}"
-      cifmw_update_containers_openstack: true
       cifmw_run_test_role: test_operator
       cifmw_run_tempest: true
-      cifmw_test_operator_tempest_registry: "{{ content_provider_registry_ip }}:5001"
+      cifmw_test_operator_tempest_registry: "{{ content_provider_os_registry_url | split('/') | first }}"
```
Comment (marios) [inline zuul.d/job.yaml:24]: just noticed, we had port before is it no longer required here? 
```
:5001
```

File: zuul.d/job.yaml [hunk 4]
```diff
@@ -20,29 +18,24 @@
...
+      cifmw_test_operator_tempest_namespace: "{{ content_provider_os_registry_url | split('/') | last }}"
       cifmw_test_operator_tempest_image_tag: "{{ content_provider_dlrn_md5_hash }}"
       cifmw_test_operator_tempest_include_list: |
         tempest.scenario.test_network_basic_ops.TestNetworkBasicOps
       cifmw_run_tobiko: true
-      cifmw_test_operator_tobiko_registry: "{{ content_provider_registry_ip }}:5001"
+      cifmw_test_operator_tobiko_registry: "{{ cifmw_test_operator_tempest_registry }}"
```
Comment (marios) [inline zuul.d/job.yaml:30]: sanity check same, port?

**Review by marios — COMMENTED** [review]
sanity check otherwise lgtm/ready to approve

File: zuul.d/job.yaml (hunk 3 above)
Comment (raukadah) [reply zuul.d/job.yaml:24]: @marios correct, It is no longer required. From successful result, https://github.com/openstack-k8s-operators/tcib/pull/179#issuecomment-2188280866.
 We can see meta content provider will return `content_provider_os_registry_url` to the [edpm zuul inventory](https://logserver.rdoproject.org/79/179/a0db69a8e875ce5e3454722e3c61cf6f16101805/github-check/tcib-podified-multinode-edpm-deployment-crc/5940ff6/zuul-info/inventory.yaml).
```
content_provider_os_registry_url: 38.102.83.3:5001/podified-antelope-centos9
```
So we no longer need to provide port here.

Comment (marios) [reply zuul.d/job.yaml:24]: ack thanks

**Review by marios — APPROVED** [review]
lgtm

**Pull request merged**